README.md

```diff
@@ -3,32 +3,37 @@
 aiXplain Pipelines enables python programmers to add AI functions
 to their software.
 
+An aiXplain pipeline is a directed acyclic graph (DAG) of AI functions built using aiXplain's designer UI. An AI function is a data processing step that relies on a machine learning model to execute. An example of an AI function is speech recognition or machine translation. It helps you process your data by calling a series of functions as defined in the DAG, abstracting the orchestration by providing a simple python function call.
```

Comment:
The text is clear to technical readers. Perhaps Dan can help us in simplifying the text to be clear also for non-technical readers.

Reply:
The text reads fine to me. But maybe "abstracting the orchestration" may not translate well for non-English speakers, maybe changing this to something different? (not sure what)